app/views/users/notes/_form.html.slim

```diff
@@ -1,3 +1,6 @@
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/simplemde/latest/simplemde.min.css">
+<script src="https://cdn.jsdelivr.net/simplemde/latest/simplemde.min.js"></script>
```

Comment:
Javascript and styles should be in the head or at the bottom of the body. What do you think about moving these styles to the head? javascript code and styles will be loaded only once and cached in the memory.